eks/cluster-loader/clusterloader2/addon.go

```diff
@@ -93,10 +93,19 @@ func (c *ClusterLoader) buildConfigMapData() (map[string]string, error) {
 		bodyString := strings.ReplaceAll(string(body), "\n", "\n    ")
 		dataMap[path.Base(uri)] = bodyString
 	}
+	if len(c.Config.Spec.ClusterLoader.TestOverrides) > 0 {
+		overrides := strings.Join(c.Config.Spec.ClusterLoader.TestOverrides, "\n    ")
+		overrides = strings.ReplaceAll(overrides, "=", ": ")
```

Comment:
Why not just have the user specify the correct value?

Reply:
Will change.